src/ontology/pato-edit.obo

```diff
@@ -22136,6 +22136,61 @@ xref: ILX:0770100
 is_a: PATO:0070033 ! neuron projection quality
 created_by: http://orcid.org/0000-0001-7258-9596
 
+[Term]
+id: PATO:0070036
+name: dendrite quality
+comment: A morphology quality inhering in a neuron by virtue of the bearer's dendrite having high or low density of classical spines. {xref="ILX:0103022"}
```

Comment:
1. It seems to me that `dendrite quality` should have `dendrite morphology` as a subclass, since there are non-morphological qualities of dendrites, and all morphological qualities of dendrites are qualities of dendrites. But perhaps there is something I'm overlooking here. 
2. The definition (adapted from ILX) doesn't seem correct to me. It basically states that dendrite morphology inheres because of some density of dendritic spines. It is trivially true that all dendrites have some density of dendritic spines (even aspiny dendrites have a density of spines -- it's zero), so it's not false to assert this. However, there are other structures, aside from dendritic spines, that are relevant here. `tufted` and `varicose` (subclasses) are dendrite morphologies that do not inhere in a neuron by virtue of the bearer's dendrite density, for example. 

Perhaps: "A morphological quality inhering in a neuron's dendrites by virtue of the bearer's physical composition." This seems _very_ general, though, so maybe some variation of this?

Reply:
I wonder if that's the reason why there seems to be entries that mean the same thing but asserted differently? from comment in #55 

> Sparsely spiny dendrite quality and spiny low density seems to mean the same thing, here are their respective ILX defs:
> Sparsely spiny dendrite quality: 'Characterized by a low density of dendritic spines.'
> Spiny low density: 'Characterized by a low density of spines on dendrites'
> Same for Spiny high density and Spiny dendrite quality.

Also agree with the second point, though I would like to get some feedback from @tgbugs on this. 

I do think having dendrite morphology and asserting `tufted` and `varicose` under that would be a good idea.

Reply:
1. Yes. Dendrite quality could a number of things beyond morphology including electrical properties of the dendrite membrane. I also currently have spiny phenotype modelled as subClassOf dendrite phenotype not as subClassOf dendrite morphological phenotype since in principle they should be treated as orthogonal.
2. Yes, that InterLex definition is ancient and incorrect. I would not limit the definition to morphological properties as mentioned in 1.

With regard to spine density, I use intersectionOf spiny and sparse like so (reusing PATO and and a phenotype modifier property, though I have not fully worked out whether the hasPhenotypeModifier approach works out as desired).

```ttl
owl:intersectionOf (
        [ a owl:Restriction ;
            owl:onProperty ilxtr:hasDendriteMorphologicalPhenotype ;
            owl:someValuesFrom ilxtr:SpinyPhenotype ]
        [ a owl:Restriction ;
            owl:onProperty ilxtr:hasPhenotypeModifier ;
            owl:someValuesFrom PATO:0001609 ] )

```

Reply:
Thanks @tgbugs, will fix accordingly. 
On the spine density, am I right to say that you would suggest removing `spiny high density` and `spiny low density` and just leaving 'spiny dendrite quality' that could be modified accordingly?

Reply:
Yes. My suggestion would be to have spiny as a quality which would imply that there is at least one spine, use complement of for aspiny (zero spines), and then use other pato qualities to provide addition information about what "more than one" means. The reason for this is that there are many different possible ways to quantify spinyness and creating an individual quality for each of them is likely to be a pain down the line.